lib/vdsm/virt/livemerge.py

```diff
@@ -402,6 +410,21 @@ def _start_commit(self, drive, job):
             self._untrack_job(job.id)
             raise exception.MergeFailed(str(e), job=job.id)
 
+    def _base_needs_prune_bitmaps(self, drive, job, base_info):
+        if base_info['format'] != 'COW':
+            log.debug("Base volume does not support bitmap operations "
+                      "job=%s drive=%s volume=%s",
+                      job.id, job.drive, job.base)
+            return False
+
+        if job.top == drive.volumeID:
```

Comment:
This should use:

    if job.active_commit:

to match other checks like _base_needs_extend().

Reply:
Ah, I see, it stores the same condition in the `Job` class. Good. I'll use it.